Security-Operations/L1-Junior-Security-Analyst.md

```diff
@@ -31,22 +31,22 @@
 <summary>Examples</summary>
 
 - I shared a demo of my work in a weekly show-and-tell session.
-- I applied a pattern I learned from training and I asked a senior analyst on my team to review it for me to ensure I applied it correctly.
-- I shadowed a senior analyst during an incident to become more familiar with the incident response process.
+- I applied a pattern I learned from training, and I asked a senior analyst on my team to review it to ensure I applied it correctly.
+- I shadowed a senior analyst during an incident to become more familiar with our Security Incident Response Plan (SIRP).
 
 </details>
 
 ## 🧭 Culture and Leadership
 
 - I show independent initiative and problem-solving.
 - I demonstrate a collaborative attitude.
-- I take responsibility for my own work.
+- I take responsibility for my work.
```

Comment:
I see grammarly at work here :)

Reply:
Just checking - would you prefer I re-add `own` here or leave it as is?

Reply:
No, leave it out. :) General rule - less words are often better.